src/components/TeamMemberComponent.js

```diff
@@ -0,0 +1,90 @@
+import React, { useState } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import {
+  Grid, Typography, Dialog, Card, CardActionArea, CardMedia, CardActions
+} from '@material-ui/core';
+import data from "../assets/TeamMembersData.json";
+import styles from "./TeamMembersLayout";
+const useStyles=makeStyles((theme)=>(styles(theme)));
+const team = data.members;
+
+export default function TeamMemberComponent() {
+  const classes = useStyles();
+  const [open, setOpen] = useState(false);
+  const [index, setIndex] = useState(0);
+  const unknown = "Unknown";
+  const [phone, setPhone] = useState(unknown);
+  const [email, setEmail] = useState(unknown);
+  const [github, setGithub] = useState(unknown);
+  const [linkedin, setLinkedin] = useState(unknown);
+  const handleOpen = (data) => {
+    setOpen(true);
+    if (data != undefined) {
+      setIndex(data);
+      if (team[data].phonenNo !== undefined)
+        setPhone(team[data].phonenNo);
+      if (team[data].emailid !== undefined)
+        setEmail(team[data].emailid);
+      if (team[data].githublink !== undefined)
+        setGithub(team[data].githublink);
+      if (team[data].linkedin !== undefined)
+        setLinkedin(team[data].linkedin);
+    }
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+    setPhone(unknown);
+    setEmail(unknown);
+    setGithub(unknown);
+    setLinkedin(unknown);
+  };
+
+  return (
+    <div>
+      <Card>
+        <Typography variant="h3" className={classes.heading}>our team</Typography>
+        <div className={classes.root}>
+          <Grid container spacing={4} className={classes.gridContainer}>
+            {team.map((tile, i) => (
```

Comment:
directly map using {team.map(teamMember => (..
instead of using an index variable

Reply:
Yeah i am mapping in the same way just the name difference